index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>SergSkol Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  </head>
+  <body>
+    <header class="header">
+        <a class="logo" href="#">
+          Serg Skol
+        </a>
+        <nav class="menu">
+            <a class="menu_icon fa-solid fa-bars" title="Menu"></a>
+        </nav>
+    </header>
+    <section class="headline">
+      <br><br><br>
```

Comment:
- [Optional] Consider using margin and padding in your CSS for spacing rather than `<br>` tags. This is best practice.